public/javascripts/continuousPlay.js

```diff
@@ -4,7 +4,8 @@
 let pageNum = parseInt(localStorage.getItem("pageNum"));
 let pageCounter = document.getElementById("page-counter");
 let nextButtonText = document.getElementById("button-next");
-let userGuessData = JSON.parse(localStorage.getItem("userGuessData")) || {};
+let allUserGuessData = JSON.parse(localStorage.getItem("userGuessData")) || [];
+let currentUserGuessData = {};
```

Comment:
Related to the array push or unshift comment below, this variable could be removed entirely and just push/unshift the object with key values like this:

```
allUserGuessData.push({ gameName: 'asdf', imgLink: 'asdf' })
```